kernel/async/monitor-common/async/Place.hh

```diff
@@ -93,17 +94,18 @@ public:
 protected:
   void pushPrivate(Tasklet* msg) {
     ASSERT(isLocal());
-    //MLOG_DETAIL(mlog::async, this, "push private", msg);
-    queue.pushPrivate(msg);
+    ASSERT(msg);
+    MLOG_DETAIL(mlog::async, this, "push private", msg);
+    queue.pushPrivate(*msg);
   }
 
   void wakeup() { mythos::lapic.sendIRQ(apicID, 32); }
 
 protected:
   size_t apicID; //< for wakeup signals
   std::atomic<bool> nestingMonitor;
-  TaskletQueue queue; //< for pending tasks
-  char padding[64-sizeof(apicID)-sizeof(nestingMonitor)-sizeof(queue)]; // TODO to ensure separate cache lines
+  TaskletQueueImpl<TaskletQueueBaseAligned> queue; //< for pending tasks
+  char padding[(sizeof(apicID)-sizeof(nestingMonitor)-sizeof(queue)) % 64]; // TODO to ensure separate cache lines
```

Comment:
yes, should work well without padding. I think, that alignment attributes are more effective and easier to use.